### ISSUE TITLE
Trezor Suite with Tails

### ISSUE THREAD

**Issue by Hercules1122** [trezor/trezor-suite#6750]
Does anyone get the Trezor working on another OS than Windows?

On Tails doesn't work.

On Whonix I run on Terminal in the folder sudo dpkg -i trezor-bridge_2.0.27_amd64.deb and sudo dpkg -i trezor-udev_2_all.deb.
Than open Suite and says ' Trezor Bridge not running '.
I got a notification though that a new Version of Suite is available, but I dont get access to Tor when I want to connect at the switch.
The Download starts.
Tor Browser on Whonix also doesnt show up http://127.0.0.1:21325/ the Trezor device.
So the Device doesnt get recognized on Whonix as well... and I dont get connection to Tor on Suite on Whonix. 

On Whonix I can start the App 'MyCrypto' and I get connection to the Ethereum Network.
Which doesn't work on Tails actually.
Than I get asked ' How would you like to connect to your Wallet? '
There is Trezor listed. ' Connect to Trezor ' - no device connected.
As the device doesnt get recognized on Whonix.

On Kali Linux, I see the http://127.0.0.1:21325/status/ page, but says connected Devices: 0
Also on Suite App it says ' Connect your Trezor '.

I would like to have the Trezor running on a Tor by default OS or Tor Browser.

**Comment by Hercules1122** [comment]
For sure on Tails the Trezor works for Electrum. 
But what about others coins? 

Just to have it running on Tor Browser would be the nicest option.

**Comment by sime** [comment]
Works plenty fine on classic Linux installations. As well as OSX and Android (Chrome Browser).

**Comment by Hercules1122** [comment]
Thanks for your reply. What about a Tor by default OS any success?

**Comment by Hercules1122** [comment]
... also do I have to do more than " run on Terminal in the folder sudo dpkg -i trezor-bridge_2.0.27_amd64.deb and sudo dpkg -i trezor-udev_2_all.deb " ... because on Kali Linux it doesn't recognize the Trezor.

**Comment by Hercules1122** [comment]
> ... also do I have to do more than " run on Terminal in the folder sudo dpkg -i trezor-bridge_2.0.27_amd64.deb and sudo dpkg -i trezor-udev_2_all.deb " ... because on Kali Linux it doesn't recognize the Trezor.

any suggestions?

**Comment by sime** [comment]
That should do it.

**Comment by Hercules1122** [comment]
Thanks for your support sime!
Made every step like here: https://trezor.io/learn/a/tor-in-trezor-suite-app 
Also dpkg trezor bridge and udev on Kali Linux. 
Status page is shown up, but connected device still 0. 
In Terminal, trezord:
trezord v2.0.27 is starting.
https: listen tcp 127.0.0.1:21325: bind: address already in use

Hmm.. it works on Windows. But didn't get Trezor running either on Suite App or Tor Browser yet on Tails, Whonix and Kali Linux.

**Comment by bosomt** [comment]
@Hercules1122 
can you please retest it with version 2.0.30 ?
Please see https://github.com/trezor/trezor-suite/issues/5131

You can download it here https://github.com/trezor/data/tree/master/bridge/2.0.30

**Comment by brianddk** [comment]
I got Bridge working under Tails many (many) years ago

https://github.com/brianddk/trezor-tails, https://github.com/brianddk/trezor-tails/wiki

IIRC, it required some IPTables entries to open up localhost to amnesia for Bridge to work.  There is also some work relating to getting the PKG dependencies and the like, but the critical bit was the IPTables

https://github.com/brianddk/trezor-tails/blob/scratch/assets/localhost.sh

I also HIGHLY suggest you run everything on obscure bridges.  I found that even the best intentioned projects usually leave some CDN dependencies that route through cloudflare that will squash TOR traffic breaking CDN dependencies (if there are any left).
